app/models/note.rb

```diff
@@ -0,0 +1,5 @@
+class Note < ApplicationRecord
+  belongs_to :user_challenge
+
+  broadcasts_to ->(note) { "notes" }, inserts_by: :prepend
```

Comment:
can you explain what this does?

Reply:
Basically, when you create a new 'note', it will be prepended to a 'div(<%= turbo_frame_tag Note.new %>)' in real-time

Reply:
interesting, I didn't know you could do that in the model itself; I've done it in a view file before ([for example](https://github.com/adrianHards/hotwire-reminder/blob/main/app/views/tasks/create.turbo_stream.erb)) though I'm assuming it achieves the same thing when it might not.

Reply:
I think the big difference is that turbo_stream can be only real-time for your side but not other's servers.
but the broadcast will allow both sides to be real-time updates!